src/styles/block/catalog.scss

```diff
@@ -0,0 +1,15 @@
+.container {
+  --colum-count: 1;
```

Comment:
There appears to be a typo in the custom property `--colum-count`. According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), custom properties should be correctly spelled. It should be `--column-count`.